imagemounter/dependencies.py

```diff
@@ -0,0 +1,153 @@
+from imagemounter import _util
+
+
+class Dependency(object):
+    installed_explanation = "INSTALLED {!s}"
+
+    def __init__(self, name, package="", why=""):
+        self.name = name
+        self.package = package
+        self.why = why
+
+    @property
+    def status(self):
+        if self.check():
+            return self.installed_explanation.format(self)
+        else:
+            return "MISSING   {!s:<20}".format(self) + self.missing_explanation
+
+
+class CommandDependency(Dependency):
+
+    def __str__(self):
+        return self.name
+
+    def check(self):
+        """Check if the command is available on the system"""
+        return _util.command_exists(self.name)
+
+    @property
+    def missing_explanation(self):
+        if self.why and self.package:
+            return "needed for {}, part of the {} package".format(self.why, self.package)
+        elif self.why:
+            return "needed for {}".format(self.why)
+        elif self.package:
+            return "part of the {} package".format(self.package)
+        else:
+            return ""
+
+
+class PyModuleDependency(Dependency):
+
+    def __str__(self):
+        # Fall back to name if not provided (in case it's the same as the package)
+        return self.package or self.name
```

Comment:
I think this method can be put in the superclass and omitted from both children.

(Doing that would list the package names for system packages instead of their commands. Probably needs rephrasing of the ``missing_explanation`` to include the command it checked for.)